pkger/parser_models.go

```diff
@@ -1044,10 +1044,11 @@ func (c colors) valid() []validationErr {
 type query struct {
 	Query  string `json:"query" yaml:"query"`
 	params []*references
+	task   []*references
 }
 
 func (q query) DashboardQuery() string {
-	if len(q.params) == 0 {
+	if len(q.params) == 0 && len(q.task) == 0 {
```

Comment:
do task options even come up on dashboard queries?  does this need to be here?

Reply:
seems like this might be a poorly named function. we can look into deeper as greg gets more context.